html/themes/custom/itchefer/itchefer.theme

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/**
+ * @file
+ * The primary PHP file for the Itchefer theme.
+ */
+
+/**
+ * Implements hook_preprocess_input().
+ *
+ * Remove news-topic for everyone but administrators.
+ */
+function itchefer_preprocess_input(&$variables) {
+  if ($variables['element']['#type'] === 'radio' && $variables['element']['#id'] === 'edit-field-topic-type-3') {
+    if (!str_contains(implode($variables[user]->getRoles()), 'administrator')) {
```

Comment:
This should probably use https://api.drupal.org/api/drupal/core%21modules%21user%21src%21Entity%21User.php/function/User%3A%3AhasRole/8.2.x or (maybe even better) check for a specific permission using https://api.drupal.org/api/drupal/core%21modules%21user%21src%21Entity%21User.php/function/User%3A%3AhasPermission/8.2.x.